src/redux/books/books.js

```diff
@@ -1,10 +1,10 @@
-const ADDBOOK = 'ADDBOOK'; // Define a constant for addBook action types
-const REMOVEBOOK = 'REMOVEBOOK'; // Define a constant for removeBook action types
+const ADD_BOOK = 'ADD_BOOK'; // Define a constant for addBook action types
+const REMOVE_BOOK = 'REMOVE_BOOK'; // Define a constant for removeBook action types
```

Comment:
- [ ] Kindly follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for your action names. Like the previous reviewer pointer out it should be `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`. Take note that you have to add the name of your repo (bookstore) and the file in which the action is found (books)

Reply:
![image](https://user-images.githubusercontent.com/13853450/178807397-5989a883-d7da-4043-9f88-3609c503f7c9.png)
tnx changed.